sklearn/sklearn-template/template/README.md

```diff
@@ -0,0 +1,105 @@
+# Template for training a Scikit-learn Model on Cloud ML Engine
+
+
+This template is designed for building a scikit-learn-based machine learning trainer that can be run on 
+Cloud ML Engine (CMLE) at scale. Before you jump in, let’s cover some of the different tools you’ll be using to get 
+your job running on CMLE.
+
+- [Google Cloud Platform](https://cloud.google.com/) lets you build and host applications and websites, store data, 
+and analyze data on Google's scalable infrastructure.
+- [Cloud Machine Learning Engine](https://cloud.google.com/ml-engine/) is a managed service that enables you to easily build 
+machine learning models that work on any type of data, of any size.
+- [Google Cloud Storage](https://cloud.google.com/storage/) (GCS) is a unified object storage for developers and 
+enterprises, from live data serving to data analytics/ML to data archiving.
+- [Cloud SDK](https://cloud.google.com/sdk/) is a command line tool which allows you to interact with 
```

Comment:
We should probably mention that this SDK provides CLI tools such as the `gcloud` command line tool for interacting with Google Cloud products and services.

Reply:
Modified.